packages/queries/src/query.test.ts

```diff
@@ -54,6 +43,31 @@ describe('queries', () => {
         };
     });
 
+    test('Find one actor with `name()`', () => {
+        const q = from('actor')
+            .columns('actor_id', 'first_name')
+            .name('findActor')
+            .one();
+
+        const result = fakeQueryResult(q);
+
+        result satisfies { actor_id: number; first_name: string };
+    });
+
+    test('Find one actor with `param()`', () => {
+        const q = from('actor')
+            .columns('actor_id', 'first_name')
+            .where({
+                actor_id: param('id', 1),
```

Comment:
ok I just realised we should switch the order of these arguments: given that we might be able to auto-generate the `id` it shuold be param(1) in case it's auto generated and param(1,'id') while we can't auto-generate it.